lib/vdsm/storage/lvm.py

```diff
@@ -529,42 +529,42 @@ def _updatepvs_locked(self, pvs_output, pv_names):
                 log.error("Missing pv: %s in vg: %s", pv.uuid, pv.vg_name)
                 continue
             self._pvs[pv.name] = pv
-            updatedPVs[pv.name] = pv
+            updated_pvs[pv.name] = pv
 
-        # Remove stalePVs
-        stalePVs = [name for name in (pv_names or self._pvs)
-                    if name not in updatedPVs]
-        for name in stalePVs:
+        # Remove stale PVs
+        stale_pvs = [name for name in (pv_names or self._pvs)
+                     if name not in updated_pvs]
+        for name in stale_pvs:
             if name in self._pvs:
                 log.warning("Removing stale PV %s", name)
                 del self._pvs[name]
 
-        return updatedPVs
+        return updated_pvs
 
     def _update_stale_pvs_locked(self, pv_names):
         """
         Check if any PV is stale after an LVM command and make it Unreadable.
         Log a warning if any PV has been made Unreadable.
         Must be called while holding the lock.
         """
-        updatedPVs = {}
-        pvNames = pv_names if pv_names else self._pvs
-        for p in pvNames:
+        updated_pvs = {}
+        pv_names = pv_names if pv_names else self._pvs
```

Comment:
Before this change pVNames was a new variable, now it replaces pvNames. Since pvNames is either list of names or a dict, this may change pv_names to a dict. This works for iteration but may break the code in the rest of function.

It is better to avoid such manipulations, keep pv_names as list of strings, and use another name for the itetarion.

An easy way to avoid this would be:

   for p in (pv_names or self._pvs):